controllers/reconcilers/configuration/configuration_reconciler.go

```diff
@@ -228,6 +230,18 @@ func (r *Reconciler) Reconcile(ctx context.Context, cr *v1.Observability, s *v1.
 		overrideLastSync = true
 	}
 
+	// Force resync if any CRs missing post-migration
+	if s.Migrated {
+		var missingCR bool
+		missingCR, err = r.checkForMissingCR(ctx, cr)
+		if err != nil {
+			return v1.ResultFailed, errors2.Wrap(err, "error checking for migrated CR")
+		}
+		if missingCR {
+			overrideLastSync = true
+		}
```

Comment:
@vmanley , @pb82 - does this need to consider the value of `.spec.descopedMode.enabled`? I'm seeing very frequent resyncs with descopedMode and only a Prometheus CR is present.

Reply:
@MikeEdgar Sounds like that would cause an issue. I'll look into it